### PR TITLE
MA 2 - integration testing

### LINKED ISSUE #2: Create api test project
Add test case to verify that
`GET /` returns "Hello World!"

### PR DESCRIPTION
**Pull request by ServAlex**
created solution and xunit project
added integration test that compares api root output to "TEST" environment variable value

### CODE REVIEW

File: src/unitTests/ApiTests.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        public ApiTests(WebApplicationFactory<minapi.Startup> factory)
+        {
+            _factory = factory;
+            System.Environment.SetEnvironmentVariable("TEST", "Hello World!");
+        }
+
+        [Fact]
+        public void Successful()
+        {
+            Assert.True(true);
```
Comment (randomdima) [inline src/unitTests/ApiTests.cs:21]: image

Comment (randomdima) [inline src/unitTests/ApiTests.cs]: image

File: src/unitTests/ApiTests.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        }
+
+        [Fact]
+        public void EnvironmentVariableValue()
+        {
+            var envVariableValue = Environment.GetEnvironmentVariable("TEST");
```
Comment (randomdima) [inline src/unitTests/ApiTests.cs]: Configuration in not testable =)

File: src/unitTests/ApiTests.cs (hunk 1 above)
Comment (ServAlex) [reply src/unitTests/ApiTests.cs]: helped to setup unit tests, in vscode, may be removed now

File: src/unitTests/ApiTests.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            Assert.Equal("Hello World!", envVariableValue);
+        }
+
+        [Fact]
+        public async Task GetHelloWorld()
```
Comment (randomdima) [inline src/unitTests/ApiTests.cs]: Use test naming convention.
https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices

File: src/unitTests/ApiTests.cs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        public ApiTests(WebApplicationFactory<minapi.Startup> factory)
+        {
+            _factory = factory;
+            System.Environment.SetEnvironmentVariable("TEST", "Hello World!");
+        }
+
+        [Fact]
+        public async Task GetHelloWorld()
+        {
+            var client = _factory.CreateClient();
```
Comment (randomdima) [inline src/unitTests/ApiTests.cs]: client should be disposed

**Review by randomdima — APPROVED** [review]
LGTM

**Pull request merged**